### PR TITLE
update seed done

### PR DESCRIPTION
**Pull request by Ragansu**
updated seed system

default main seed  = 31415

default for Starting kit = 31415

default of run_ingestion.py (used by codabench) = UTC current time in secs can be changed to a fixed seed from parcer input
(or metadata.yaml for codabench)

Thank You

RC

### CODE REVIEW

File: ingestion_program/ingestion_parallel.py [hunk 1]
```diff
@@ -403,7 +404,8 @@ def predict_submission(self, test_settings):
         # create a product of set and test set indices all combinations of tuples
         all_combinations = list(product(set_indices, test_set_indices))
         # randomly shuffle all combinations of indices
-        np.random.shuffle(all_combinations)
+        random_state_initial = np.random.RandomState(initial_seed)
```
Comment (ihsaan-ullah) [inline ingestion_program/ingestion_parallel.py:407]: use same name for random state `random_state` as in ingestion

**Review by ihsaan-ullah — APPROVED** [review]
everything looks good

File: ingestion_program/ingestion_parallel.py (hunk 1 above)
Comment (Ragansu) [reply ingestion_program/ingestion_parallel.py:407]: They should be same. because Its copy paste. the difference is in ingestion_parallel its in 2 different places 
which in ingestion.py its next to each other like there is 2 random state

File: ingestion_program/ingestion_parallel.py [hunk 2]
```diff
@@ -383,7 +384,7 @@ def fit_submission(self):
         print("[*] Calling fit method of submitted model")
         self.model.fit()
 
-    def predict_submission(self, test_settings):
+    def predict_submission(self, test_settings, initial_seed=31415):
```
Comment (cjh1) [inline ingestion_program/ingestion_parallel.py:387]: We should probably pull 31415 out as a constant as its used in two places.

Comment (Ragansu) [reply ingestion_program/ingestion_parallel.py:387]: So we use a different number ?

Comment (Ragansu) [reply ingestion_program/ingestion_parallel.py]: Please check now

**Review by cjh1 — APPROVED** [review]

**Comment by Ragansu** [comment]
Thank You @ihsaan-ullah  and @cjh1
